### PR TITLE
Update readme.md requirements

### PR DESCRIPTION
**Pull request by DplusG**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ❌
| Breaks BC?    | ❌
| Fixed issues  | #40

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,6 +20,7 @@ The package provides an ability to record performance profiles.
 ## Requirements
 
 - PHP 7.4 or higher.
+- If in `yiisoft/aliases` configuration will be no alias `@runtime`, then application throws error `Invalid path alias: @runtime/profiling/{date}-{time}.txt`
```
Comment (samdark) [inline README.md:23]: That should be in installation section. That's not a requirement but an installation step.

**Review by samdark — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -29,6 +29,8 @@ The package could be installed via composer:
 composer require --dev yiisoft/profiler --prefer-dist
 ```
 
+- If in `yiisoft/aliases` configuration will be no alias `@runtime`, then application throws error `Invalid path alias: @runtime/profiling/{date}-{time}.txt`
```
Comment (vjik) [inline README.md:32]: We should describe about `@runtime` in params and suggest change it (how and where) if need.

**Review by vjik — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -29,6 +29,24 @@ The package could be installed via composer:
 composer require --dev yiisoft/profiler --prefer-dist
 ```
 
+If in `yiisoft/aliases` configuration will be no alias `@runtime`, then application throws error `Invalid path alias: @runtime/profiling/{date}-{time}.txt`
+
+Than install the following package `yiisoft/aliases` via composer:
```
Comment (samdark) [inline README.md:34]:
```suggestion
If there will be no `@runtime` alias in `yiisoft/aliases` configuration defined, application will throw "Invalid path alias" error.

To solve it, install the following package `yiisoft/aliases` via composer:

```

**Review by samdark — APPROVED** [review]

**Comment by samdark** [comment]
Thanks!

**Pull request merged**